blockchain/tx_pool.go

```diff
@@ -511,6 +520,11 @@ func (pool *TxPool) SetGasPrice(price *big.Int) {
 	}
 }
 
+// only for test
+func (pool *TxPool) SetBaseFee(baseFee *big.Int) {
```

Comment:
Just for test. To test changing the `gasPrice` in txPool we need the setter like it because we added a guard if hardforked or not in legacy function, `SetGasPrice`

Reply:
No, I meant why did you place it here, not into a test file.

Reply:
It needs to set the `gasPrice` field in txPool.

Reply:
I'm not asking about its functionality, asked where the test function is located.

Reply:
Are you asking where is the location to use `SetBaseFee`?

Reply:
The second one. The function can be located in `tx_pool_test.go` and there's no constraint to be located in `tx_pool.go`. If you think it's trivial, how about skipping it?